helm/vertical-pod-autoscaler-app/templates/crds/ciliumnetworkpolicy.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if .Values.ciliumNetworkPolicy.enabled }}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: {{ include "vertical-pod-autoscaler.crds.fullname" . }}
+  annotations:
+  labels:
+    {{- include "vertical-pod-autoscaler.crds.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+specs:
+  - description: "Allow the Job to access the K8s API Server"
+    endpointSelector:
+      # Since we are excluding the job-name from the matched resources from cilium identity evaluation
+      # we simply can't use it, because the policy will never be matched against the job
+      # If and when this will be reverted: https://github.com/giantswarm/cilium-app/pull/80/files#diff-19d6b2fc3287f6aa81dac49278df30124546319c557f738958b6f4b1326e4867R1422
+      # we can use the job name once again
+      matchLabels: {}
```

Comment:
This one is trickier since the upstream chart has no label defined here https://github.com/cowboysysop/charts/blob/6bbde64075b77a642be1492bf0134c95beccc628/charts/vertical-pod-autoscaler/templates/crds/job.yaml#L17

Reply:
Upstream issue https://github.com/cowboysysop/charts/issues/526

Reply:
@silenteh The upstream issue is now merged in 9.1.0 so we can fix it the same way as https://github.com/giantswarm/vertical-pod-autoscaler-app/pull/201#discussion_r1301177082